recipes/natural_language_processing/rag-nodejs/bootc/Containerfile

```
@@ -0,0 +1,48 @@
+# Example: an AI powered sample application is embedded as a systemd service
+# via Podman quadlet files in /usr/share/containers/systemd
+#
+# from recipes/natural_language_processing/rag-nodejs, run
+# 'make bootc'
+
+FROM quay.io/centos-bootc/centos-bootc:stream9
+
+ARG SSHPUBKEY
+
+# The --build-arg "SSHPUBKEY=$(cat ~/.ssh/id_rsa.pub)" option inserts your
+# public key into the image, allowing root access via ssh.
+RUN set -eu; mkdir -p /usr/ssh && \
+    echo 'AuthorizedKeysFile /usr/ssh/%u.keys .ssh/authorized_keys .ssh/authorized_keys2' >> /etc/ssh/sshd_config.d/30-auth-system.conf && \
+    echo ${SSHPUBKEY} > /usr/ssh/root.keys && chmod 0600 /usr/ssh/root.keys
+
+ARG RECIPE=rag-nodejs
+ARG MODEL_IMAGE=quay.io/ai-lab/granite-7b-lab:latest
+ARG APP_IMAGE=quay.io/ai-lab/${RECIPE}:latest
+ARG SERVER_IMAGE=quay.io/ai-lab/llamacpp_python:latest
+ARG CHROMADBImage=quay.io/ai-lab/chromadb
```

I dont believe this is correct, should be: 
```suggestion
ARG CHROMADB_IMAGE=quay.io/ai-lab/chromadb:latest
```

---

Its a direct copy from the existing rag recipe - https://github.com/containers/ai-lab-recipes/blob/7fbdf03a8dadbf5a41da283fdc1c07ea8d1ec61a/recipes/natural_language_processing/rag/bootc/Containerfile#L21

---

Updated